<?xml version="1.0"?>
<launch>

	<rosparam command="load" file="$(find ar_navigation)/qr_positions/qr_positions.yaml"/>

	<!-- qr_saver node -->
	<node pkg="ar_navigation" type="qr_saver" name="qr_saver" output="screen"/>
	
</launch>

